src/Internal/Int64.elm

```diff
@@ -0,0 +1,165 @@
+module Internal.Int64 exposing (Int32s, Int64, config, fromInt32s, toInt32s, zero)
+
+import Bitwise
+import Bytes exposing (Endianness(..))
+import Internal.IntType as IntType
+
+
+{-| Basic Int64 data type consisting of two Int32s.
+To not bloat this package, only operations needed for en- and decoding are defined.
+-}
+type Int64
```

Comment:
Can we separate the `Int64` from the ops implementation? For example by adding a `Internal.Types.Int64` module.

Reply:
Sure.

Reply:
Done. This also gets rid of the type alias solution to avoid exposing unwanted functions, since the Int64 wrapping/unwrapping functions are enough to define the various operations on the data type.